doc/usage/configuration.rst

```diff
@@ -520,25 +520,26 @@ Project information
    .. versionadded:: 0.5
 
    .. versionchanged:: 1.4
-      The default is now ``'default'``. It is similar to ``'python3'``;
-      it is mostly a superset of ``'python'``. but it fallbacks to
-      ``'none'`` without warning if failed.  ``'python3'`` and other
-      languages will emit warning if failed.  If you prefer Python 2
-      only highlighting, you can set it back to ``'python'``.
+      The default is now ``'default'``. It is similar to ``'python3'``; it is
+      mostly a superset of ``'python'``. but it fallbacks to ``'none'`` without
+      warning if failed.  ``'python3'`` and other languages will emit warning
+      if failed.  If you prefer Python 2 only highlighting, you can set it back
+      to ``'python'``.
```

Comment:
This paragraph is good example of a paragraph with originally short lines getting rewrapped to longer lines. Is there really necessity to do that? if yes I propose all such changes with no alterations in contents are strictly delimited to one commit with nothing else.